DAG-insert-elk-sirene.py

```diff
@@ -12,18 +11,15 @@
     get_colors,
     update_color_file,
 )
-from dotenv import load_dotenv
 from operators.clean_folder import CleanFolderOperator
 
-load_dotenv()
-
-DAG_FOLDER = os.getenv("DAG_FOLDER")
-DAG_NAME = os.getenv("DAG_NAME")
-TMP_FOLDER = os.getenv("TMP_FOLDER")
+DAG_FOLDER = Variable.get("dag_folder")
+DAG_NAME = Variable.get("dag_name")
+TMP_FOLDER = Variable.get("tmp_folder")
```

Comment:
I think those three variable could be in clear. No need to grow number of variables in airflow if we don't need it.